test/e2e/specs/editor/various/popovers.spec.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor state', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.beforeEach( async ( { pageUtils } ) => {
+		await pageUtils.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.describe( 'dropdown', () => {
+		test( 'toggles via click', async ( { page } ) => {
+			const isMoreMenuOpen = async () =>
+				!! ( await page.locator(
+					'.interface-more-menu-dropdown__content'
+				) );
+
+			expect( await isMoreMenuOpen() ).toBe( true );
```

Comment:
We can rewrite this in Playwright style.
```suggestion
			await expect( page.locator('role=menu[name="Options"i]') ).toBeVisible();
```

Reply:
In addition, I think the test is wrong here. The initial menu should not be visible before clicking on the toggle button.